getting-started.html

```diff
@@ -0,0 +1,149 @@
+    <p>Most videos below are just about 2 minutes long. Written documentation is linked below too, if you prefer it. And <a href="https://keybase.io/team/bisq" target="_blank">support</a> is never too far away, if you need it.</p>
+
+    <div class="admonition grey">
+        <p class="note">Before you can start trading on Bisq, you'll need a little bitcoin for a security deposit and fees (0.01 BTC should be enough). Each trader must lock bitcoin in a multisignature escrow until the trade is complete—this is part of what makes trading on Bisq highly secure.</p>
+        <p class="note"><a href="https://bisq.wiki/Funding_your_wallet#How_to_Obtain_Your_First_Bitcoin" target="_blank">Here are some tips</a> on getting this bitcoin without surrendering your data to a corporate, centralized exchange.</p>
+    </div>
+
+    <div class="start-step-container">
+        <div class="start-step">
+            <div class="step-text">
+                <h2>Download and Install Bisq</h2>
+                <p>You can download and install Bisq easily <a href="/downloads">from this website</a> or <a href="https://github.com/bisq-network/bisq/releases/latest" target="_blank">from GitHub</a>.</p>
+                <p>It's also a good idea to verify your installer file to be extra sure it hasn't been meddled with. See the videos here for directions. More details are in <a href="https://bisq.wiki/Downloading_and_installing" target="_blank">this wiki article</a>.</p>
+                <p>If you'd like to build Bisq from source, <a href="https://github.com/bisq-network/bisq/blob/master/docs/build.md" target="_blank">here are directions</a>. Bisq is free/libre open-source software that <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank">you can contribute to</a>.</p>
+            </div>
+            <div class="step-feature">
+                <div class="feature-selector"><a href="#" id="select-windows-install-video" class="install-step-item active">Windows</a><a href="#" id="select-linux-install-video" class="install-step-item">macOS or Linux</a></div>
+                <div class="responsive-youtube-container install-step-item active" id="windows-install-video">
+                    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/XABzwXw6X0A" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+                <div class="responsive-youtube-container install-step-item" id="linux-install-video">
+                    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/dTfM4AsxNHY" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+            </div>
+        </div>
+
+        <div class="start-step">
+            <div class="step-text">
+                <h2>Set Up and Tour</h2>
```

Comment:
I would rename this section to "Backup your Bisq seed" since that's probably the key action point here